<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Board Setting</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PortComboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudrateComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ConnectPushBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>64</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Servo ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SearchPushBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ServoIDLineEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>connected boards</string>
       </property>
      </widget>
      <widget class="QComboBox" name="BoardComboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="DisconnectPushBtn">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>JOG Setting</string>
      </property>
      <widget class="QPushButton" name="FwdPushBtn">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>FWD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BwdPushBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>BWD</string>
       </property>
      </widget>
      <widget class="QLabel" name="JoglocLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="HomePushBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Pos1LineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pos1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pos2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Pos2LineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>location(0~4090) :</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>321</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Test Setting</string>
      </property>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Push Count</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PushCountLineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SetupSavePushBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetupResetPushBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>460</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>231</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Actuator Setting</string>
      </property>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed Limit (1~1023)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SpeedLimitLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Temp Speed Limit (1~1023)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SpeedTempLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Limit (1~1600)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="CurrentLimitLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Temp Current Limit (1~1600)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="CurrentTempLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>BWD Limit (0~4095)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BWDLimitLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>FWD Limit (0~4095)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="FWDLimitLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Acceleration rate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="AccLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Deceleration Rate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DecLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ActuatorSaveBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>430</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ActuatorResetBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>430</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Run</string>
     </attribute>
     <widget class="QPushButton" name="SetupExitPushBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>460</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>301</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="MainLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
